spec/lib/client_summarizer_spec.rb

```diff
@@ -16,7 +16,7 @@
 
   describe "Ncr::WorkOrder" do
     it "builds summary" do
-      wo = create(:ncr_work_order, amount: 123)
+      create(:ncr_work_order, amount: 123)
```

Comment:
They are unused variables. I had the file open and my syntastic plugin was
highlighting them as unused so I thought I would get rid of them while I
was there. Would you prefer them removed in a separate PR? I could go
either way on that

On Wednesday, December 16, 2015, Peter Karman <notifications@github.com>
wrote:

> In spec/lib/client_summarizer_spec.rb
> <https://github.com/18F/C2/pull/876#discussion_r47789233>:
>
> > @@ -16,7 +16,7 @@
> >
> >    describe "Ncr::WorkOrder" do
> >      it "builds summary" do
> > -      wo = create(:ncr_work_order, amount: 123)
> > +      create(:ncr_work_order, amount: 123)
>
> why throw away the variable assignment? and why in this PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/C2/pull/876/files#r47789233>.
>